error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/FileRules.java

```diff
@@ -141,4 +142,18 @@ File after(File directory, String prefix, String suffix) throws IOException {
       return Files.createTempFile(directory.toPath(), prefix, suffix).toFile();
     }
   }
+
+  /** Prefer `File#mkdirs` before `Files#exists` to avoid concurrency issues. */
+  static final class MkdirsBeforeFilesExists {
```

Comment:
It's not very obvious, but we use a "naming algorithm" for after templates that we at some point want to automate. It's based on the content of the after template :), will push a suggestion.